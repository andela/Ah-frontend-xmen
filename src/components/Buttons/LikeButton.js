import React from 'react';

const LikeButton = (props) => {
  const classNames = props.liked ? 'btn btn-sm btn-default' : 'btn btn-sm btn-outline-default';
  return (
    <button type="button" id="like-btn" onClick={props.handleLike} className={classNames}>
      <span className="pr-1"><i className="fa fa-sm fa-thumbs-up" /></span>
      {props.likesCount}
    </button>
  );
};

export default LikeButton;
