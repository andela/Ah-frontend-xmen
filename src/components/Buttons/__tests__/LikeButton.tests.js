import React from 'react';
import { shallow } from 'enzyme';
import LikeButton from '../LikeButton';

describe('LikeButton', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<LikeButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
