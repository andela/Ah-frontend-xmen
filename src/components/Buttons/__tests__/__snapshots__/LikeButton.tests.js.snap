// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LikeButton should render without crashing 1`] = `ShallowWrapper {}`;
